docs/PULL_REQUESTS.md

```diff
@@ -0,0 +1,43 @@
+# Pull Request Guidelines
+
+Pull Requests are both a great way to maintain high quality code and an opportunity to learn from each other. Here are a few best practices to get you started:
+
+## Submitting a Pull Request
+Many times the pull request is the team's first opportunity to collaborate on an otherwise independent feature.  When creating a Pull Request, put yourself in the reviewer's seat and imagine that they have no context into the work that you've been doing.  The details that you share in your Pull Request help to build up that context facilitating crucial knowledge sharing within the team.
+
+- It's much better to submit small pull requests often. If your changes involve more than 5-10 files, consider breaking it into smaller chunks. It's much easier for a reviewer to read through a four file change than it is to read through a massive refactor.
+- Include a helpful PR description.This could include, but is not limited to:
+  - A link to a issue, ticket or other request if there is one. You can link in most source control by just noting the ticket number---#542. In other situations, drop a URL. This gives the reviewer some context.
+  - A brief explanation on why you are submitting the PR. This helps the reviewer, and also provides context if your change is referenced in the future.
+  - A description of the changes, especially if there are a lot of file changes or multiple components being changed. For example, it is helpful to point out that file 1, 2, & 3 relate to this functionality update while files 4 & 5 relate to a different one
+  - A screenshot or gif of the change. This does not prevent the reviewer from pulling down the code and verifying independently, but it does help with smaller styling changes or if the project is arduous to pull down and set up.
+  - Steps to test. This could describe how to navigate to the changed component, what steps to perform with the component, etc. Example:
+    > ### Steps To Test:
+    > - Run Storybook
+    > - Navigate to Pages > Homepage
+    > - Confirm that this thing changed
+    > - Confirm that when button is clicked, text updates
+
+## Reviewing a Pull Request
+
+- Who can review? Simple, everyone should review code. If you are new to a project, you'll get a feeling for the structure other projects. If you are experienced, you'll be able to offer more insights.
+- Each project will need to decide who is able to officially approve a request. If you are principle reviewer, try to finish a review within 24 hours. When you approve a PR, leave a comment saying it looks good or a simple (thumbs up). Some systems have official approval buttons.
+- Merging a pull request is the responsibility of the person who opened the PR. Why? There may be additional tasks such as deploying or updating a tag.
```

Comment:
This is not always the case for this repo, the responsibility will lie with members of the Developer Relations team at Sitecore. A lot of the PR's are submitted by the community and they either will not have permission to merge, or at least wont have access to the back-end system should the merge trigger a deploy,